public/js/cinode-recruitment-public.js

```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
         formData.append("linkedInUrl", linkedInUrl.val());
         formData.append("state", 0);
         formData.append("currencyId", currencyId);
-        if (pipelineId){
+
+        if (pipelineId) {
           formData.append("pipelineId", pipelineId);
+        } else if (multiplePipeline) {
+          formData.append("pipelineId", multiplePipeline);
         }
-        
-        if (pipelineStageId){
-        formData.append("pipelineStageId", pipelineStageId);
+
+        if (pipelineStageId) {
```

Comment:
`pipelineStageId` never defined anywhere, is that correct?

Reply:
pipelineStageId is defined inside form, and its fetched from backend